### ISSUE TITLE
Inline images not added to Word documents

### ISSUE THREAD

**Issue by bpasham** [erezlife/html2docx#62]
if there is a space after comma images are not added to the document
`src="data:image/jpg;base64, iVBORw0KGgoA`

this is perfectly acceptable and most often the developer doesn't have control over on removing the space in the data URL, and is perfectly valid to have space there. you have to trim the space I guess to make it more reliable.

It took me more than a day why html renders properly, pdf convertors working properly but not html2docx. See attached sample to test. 

Thanks

[mydoc.html.txt](https://github.com/erezlife/html2docx/files/6408270/mydoc.html.txt)

**Comment by francoisfreitag** [comment]
Yep, that looks like a bug. From https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs:

> As a URL, the data should be formatable with whitespace (linefeed, tab, or spaces)

I believe all characters outside of the base64 alphabet should be stripped before parsing the base64 data.

PRs with tests are welcome.
